defaults/main.yml

```diff
@@ -228,10 +228,11 @@ preseed_admin_sshkeys: [ '{{ lookup("pipe", "ssh-add -L | grep ^ssh || cat ~/.ss
 preseed_debian_locale: 'en_US.UTF-8'
 
 
-# .. envvar:: preseed_debian_keyboard_variant
+# .. envvar:: preseed_debian_keyboard_keymap
 #
 # Keyboard layout enabled during installation.
-preseed_debian_keyboard_variant: 'American English'
+# The current keymap layout can be looked up via ``setxkbmap -query``.
+preseed_debian_keyboard_keymap: 'en'
```

Comment:
Does this affect all of the keyboard internationalization questions? Also, does this work with Debian Wheezy installer?